build

```
@@ -8,6 +8,16 @@
 
 # WARNING: note harcoding of version14!
 
+# import and update enclone-data
+
+if ( !(-d enclone-data) ) then
+    git clone --depth=1 https://github.com/10XGenomics/enclone-data.git
+endif
+cd enclone-data
+git fetch --depth=1
+git checkout `cat ../data_version`
+cd ..
```

```suggestion
git -C enclone-data fetch --depth=1 `cat ../data_version`
git -C enclone-data switch --detach FETCH_HEAD
```
You must first fetch the commit that you want to check out, before you check it out.